modules/cram/templates/stragler_call.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euo pipefail
+
+convert_to_bam () {
```

Comment:
since the subworkflows PR was merged the input is always a `.bam`. we can skip the conversion and use `--regions`? or does the straglr fork not support this? in that case rename functions to `slice_bam` and `slice_bam_cleanup` and add a note that the straglr fork does not support `--bed`.

Reply:
I'll remove it. However I do not understand what you mean with the --regions? The bam conversion does not do any slicing?